onnxruntime/core/graph/contrib_ops/contrib_defs.cc

```diff
@@ -3335,6 +3335,11 @@ void RegisterContribSchemas() {
           AttributeProto::STRING,
           OPTIONAL_VALUE)
       .Attr("notes", "(Optional) Some notes for the model", AttributeProto::STRING, OPTIONAL_VALUE)
+      .Attr(
+          "max_size",
```

Comment:
nit: this attribute seems very specific to just QNN. But I can't think of anything better other than maybe a string attribute called something like "ep_metadata" with values that an EP would know how to parse like "max_size|1". But maybe that's too complicated.

This seems OK to me, just wanted to note it.